integration/experiment/launch_util.py

```diff
@@ -65,6 +65,8 @@ def launch_run(agent_conf, app_conf, run_id, output_dir, extra_cli_args,
     app_name = app_conf.name()
 
     uid = '{}_{}_{}'.format(app_name.lower(), agent_name, run_id)
+    if label is not None:
+        uid += '_{}'.format(label)
```

Comment:
This breaks parsing of the iteration # that's on the end of uid.  This needs to be revisited.